### PR TITLE
add announcement for removing Kubic support for older distros

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@vrothberg @rhatdan @TomSweeneyRedHat @ashcrow PTAL.

### CODE REVIEW

**Comment by lsm5** [comment]
need to fix the URL for package list.

**Review by ashcrow — APPROVED** [review]

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
LGTM

**Comment by lsm5** [comment]
@TomSweeneyRedHat when does this get displayed on podman.io?

File: _posts/2021-03-02-podman-support-for-older-distros.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Announcement: Support for Older Distros on Kubic Project/OBS
```
Comment (TomSweeneyRedHat) [inline _posts/2021-03-02-podman-support-for-older-distros.md:2]: This needs to change to

```
title: Announcement&#58;  Support for Older Distros on Kubic Project/OBS
```

File: _posts/2021-03-02-podman-support-for-older-distros.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+layout: default
+author: lsm5
+categories: [blogs]
+tags: containers, podman, distro, linux, centos, ubuntu, debian
+---
+![podman logo](https://podman.io/images/podman.svg)
+
+# Announcement: Support for Older Distros on Kubic Project/OBS
```
Comment (TomSweeneyRedHat) [inline _posts/2021-03-02-podman-support-for-older-distros.md:10]: Change colon here as above

**Comment by lsm5** [comment]
@TomSweeneyRedHat https://github.com/containers/podman.io/pull/363

**Pull request merged**